exporter/README.md

```diff
@@ -1,109 +1,39 @@
-A variety of exporters are available to the OpenTelemetry Service (both Agent and Collector)
+# Exporters
 
-## Collector
+A variety of exporters are available to the OpenTelemetry Service:
```

Comment:
Nit: you can also mention that opentelemetry-service-contrib has more exporters.

Reply:
That's a good point. Let's add it in a separate PR, we also need to add a related explanation on how to have a build with custom receivers/processors/exporters.

Reply:
Follow-up issue https://github.com/open-telemetry/opentelemetry-service/issues/213